boat/utils.js

```diff
@@ -75,6 +75,10 @@ const utils = {
 
   plurialify (count, word) {
     return count === 1 ? word : `${word}s`
+  },
+
+  getMemberRoles (guild, member) {
+    return guild.roles.filter(role => member.roles.includes(role.id))
```

Comment:
I don't think it matters, but it'd be more efficient to map from member roles, which will always have less entries than there are server roles.
```suggestion
    return member.roles.map(id => guild.roles.get(id))
```